packages/moonstone/Dropdown/Dropdown.js

```diff
@@ -261,6 +262,27 @@ const DropdownBase = kind({
 	},
 
 	computed: {
+		children: ({children, selected}) => {
+			return filterChildren(children).map((child, i) => {
+				const aria = {
+					role: 'checkbox',
+					'aria-checked': selected === i
+				};
+
+				if (typeof child === 'string') {
+					return {
+						...aria,
+						children: child,
+						key: `item${i}`
+					};
+				}
+
+				return {
+					...aria,
+					...child
+				};
+			});
+		},
```

Comment:
Should we apply the filter method on the `title` computed function below?

Reply:
We should. Good catch!